gateway/test/search_mgmt_test.go

```diff
@@ -1121,10 +1122,20 @@ func (s *GatewayOpsTestSuite) TestAnalyzeDocument() {
 				creds = *t.creds
 			}
 
-			resp, err := searchAdminClient.AnalyzeDocument(ctx, req, grpc.PerRPCCredentials(creds))
+			_, err := searchAdminClient.AnalyzeDocument(ctx, req, grpc.PerRPCCredentials(creds))
 			if t.expect == codes.OK {
```

Comment:
I'm a bit confused by what we're doing here. If we get OK why do we call analyze again and possibly expect it to return unavailable?